### DISCUSSION TITLE
Adminer of MariaDB LXC

### DISCUSSION THREAD

**Discussion by cryd-s** [tteck/Proxmox#1776] [Help Needed]
Hi,

i want to migrate my homeassistant maria db (2.2gb) to the maria lxc.

I have exported this with phpmyadmin from the maria db addon in homeassistant.

In the maria db lxc i cannot import such a big file via upload (max filesize 2mb) 

Someone a idea?

**Comment by tteck** [comment]
The only thing that I can think of is:

1. Open Home Assistant.
2. Navigate to the "Developer tools" section.
3. Click on "Services."
4. In the list of available services, locate "Recorder: Purge."
5. Check the "Repack" option and enable it.
6. Run the "Recorder: Purge" service.

Hopefully this will shrink the database enough.

https://www.home-assistant.io/integrations/recorder/#service-purge
